website/docs/components/table/partials/specifications/anatomy.md

```diff
@@ -2,20 +2,22 @@
 
 ### Table headers
 
-![Table header anatomy](/assets/components/table/table-header-anatomy.png)
+![Table header selection anatomy](/assets/components/table/table-header-select-anatomy.png)
 
 | Element          | Usage                                           |
 |------------------|-------------------------------------------------|
+| Checkbox         | Optional, but required when cells yield a checkbox |
 | Label            | Required                                        |
 | Sort direction   | Options: none, ascending, descending            |
```

Comment:
The "tooltip" element is missing from the table but present in the image. Let's make sure these are aligned.